python/llm/src/bigdl/llm/transformers/model.py

```diff
@@ -89,6 +89,18 @@ def from_pretrained(cls,
         optimize_model = kwargs.pop("optimize_model", True)
 
         if load_in_4bit or load_in_low_bit:
+
+            if config_dict.get("quantization_config", None) is not None:
```

Comment:
do we need to add it to Python Doc?

Reply:
Added and passing `quantization_config` is no longer required.